### PR TITLE
Make navbar dropdowns clickable links

### PR DESCRIPTION
**Pull request by VinceVal**
Clicking the dropdown will take you to the most logical option on that
menu instead of just toggling the menu open/closed.

### CODE REVIEW

**Comment by SiCKRAGEB0T** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your report has all the below criteria checked

- [x] Give a description on what the PR is for.
- [x] Make sure you send a PR against the DEVELOP branch
- [x] Don't send big changes in one go. Split up big PRs into multiple smaller (easier manageble) PRs

Please make sure you also read [contribution guide](https://github.com/SiCKRAGETV/SickRage/blob/master/contributing.md) and followed all the steps.

Thanks!

File: gui/slick/views/layouts/main.mako [hunk 1]
```diff
@@ -167,7 +167,7 @@
                         </li>
 
                         <li class="dropdown">
```
Comment (miigotu) [inline gui/slick/views/layouts/main.mako]: Can you also add `id="NAVtools"` to this list item?

**Comment by MGaetan89** [comment]
Some people have issues with the navbar on devices with touchscreen, and we add a click on the item that opens the dropdown? Is this really what we want?

**Comment by OmgImAlexis** [comment]
@MGaetan89 basically said the same thing on IRC. We need to re work the dropdowns and I think this may make it actually impossible now to use them on touchscreens as the dropdowns now can't be hovered at all.

**Comment by fernandog** [comment]
@VinceVal can you make /manage click go to /manage/ it self rather then /manage/episodeStatuses/

I also agree with @miigotu, we should go directly to errorlogs/viewlog/

**Comment by OmgImAlexis** [comment]
@VinceVal jump on IRC when you can and we'll see if we can get this all worked out.

**Comment by miigotu** [comment]
I dont think direct to /manage/ is a great idea. Why should it go right to mass update?

**Comment by fernandog** [comment]
Why dont we revert this PR as it only create an issue for touch devices?

**Comment by miigotu** [comment]
For reference, Im adding this link here and in the other PR, so everyone knows the solution we have decided on.

http://getbootstrap.com/components/#btn-dropdowns-split

**Comment by Liferose** [comment]
If we're creating split dropdowns can we restyle the buttons? The current gradient doesn't render well on mobile, or look well with the desktop glyphs.

**Comment by miigotu** [comment]
@Liferose one step at a time.

**Pull request merged**